rotkehlchen/tests/unit/globaldb/test_globaldb_cache.py

```diff
@@ -249,7 +246,7 @@ def test_curve_cache(rotkehlchen_instance, use_curve_api, globaldb):
     ))
 
     # Check the pools, coins and addresses from addressbook have been properly cleared
-    pools, gauges = read_curve_pools_and_gauges()
+    pools, gauges = get_read_curve_pools_and_gauges(ChainID.ETHEREUM)()
```

Comment:
this looks ugly. Why not make chainId and argument of `read_curve_pools_and_gauges`?

Reply:
using it as a class now because these functions are needed without any parameters in `ReloadablePoolsAndGaugesDecoderMixin` and some more places.